fdbclient/MonitorLeader.actor.cpp

```diff
@@ -483,6 +483,8 @@ ACTOR Future<Void> monitorNominee(Key key, ClientLeaderRegInterface coord, Async
 					    .detail("Hostname", hostname.get().toString())
 					    .detail("OldAddr", coord.getLeader.getEndpoint().getPrimaryAddress().toString());
 					connFile->getMutableConnectionString().resetToUnresolved();
+					// 50 milliseconds delay to prevent tight resolving loop due to outdated DNS cache
+					wait(delay(0.05));
```

Comment:
Is it possible that many processes will hit this code path (or the other in this commit) at the same time and resolve this hostname in a loop, even if delayed every 50ms? If so, that is still potentially a lot of DNS load in large clusters.

Reply:
Yes it is possible. I don't have a good estimation about how much the impact would be in that case, so I decided to use a small value to start. If we later decided this is too fast, we can use a large value or probably turn this into a knob.